numba/core/event.py

```diff
@@ -250,7 +254,8 @@ def on_start(self, event):
     def on_end(self, event):
         self._depth -= 1
         if self._depth == 0:
-            self._duration = timer() - self._ts
+            last = getattr(self, "_duration", 0)
+            self._duration = (timer() - self._ts) + last
```

Comment:
I think the docstring needs an update? This is effectively measuring the total time spent between `*START*` and `*END*` events of a given kind during the time the listener is active?

Reply:
updated in https://github.com/numba/numba/pull/6636/commits/2aa6f690a59564394de3b423e2e8df8b05240e8b